src/components/admin/clients/client-list-skeleton.tsx

```diff
@@ -0,0 +1,11 @@
+import { ClientCardSkeleton } from "@/components/admin/clients/client-card-skeleton";
+
+export const ClientListSkeleton = () => {
+  return (
+    <>
+      {Array.from({ length: 3 }).map((_, index) => (
```

Comment:
Having 3 is sometimes bouncy on the graphics - like searching for one name and it bounces to three and then delete it bounces back to 3 and then the full list, but good handling using Suspense.